### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by carusogabriel**
Found with https://github.com/vlajos/misspell-fixer

### CODE REVIEW

File: reference/ev/watcher-callbacks.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   a bitmask of received events.
  </para>
  <para>
-  Watcher callbacks are passed to the watcher contructors(the classes derived
+  Watcher callbacks are passed to the watcher constructors(the classes derived
```
Comment (nikic) [inline reference/ev/watcher-callbacks.xml]:
```suggestion
  Watcher callbacks are passed to the watcher constructors (the classes derived
```

File: reference/mssql/ini.xml [hunk 2]
```diff
@@ -47,7 +47,7 @@
      <entry></entry>
     </row>
     <row>
-     <entry>mssql.compatability_mode</entry>
+     <entry>mssql.compatibility_mode</entry>
```
Comment (nikic) [inline reference/mssql/ini.xml]: To make sure, did you check that there is no typo in the ini name of the extension?

File: reference/mysqlnd_qc/quickstart.xml [hunk 3]
```diff
@@ -1367,7 +1367,7 @@ array(4) {
   array(9) {
     ["query"]=>
     string(25) "DROP TABLE IF EXISTS test"
-    ["occurences"]=>
+    ["occurrences"]=>
```
Comment (nikic) [inline reference/mysqlnd_qc/quickstart.xml]: I'd assume that this is a typo in the implementation -- if we want to change the output here, the implementation needs to change first.

File: reference/parle/parle.parser.xml [hunk 4]
```diff
@@ -77,7 +77,7 @@
     <fieldsynopsis>
      <modifier>const</modifier>
      <type>integer</type>
-     <varname linkend="parle-parser.constants.error-unkown-token">Parle\Parser::ERROR_UNKOWN_TOKEN</varname>
+     <varname linkend="parle-parser.constants.error-unknown-token">Parle\Parser::ERROR_UNKNOWN_TOKEN</varname>
```
Comment (nikic) [inline reference/parle/parle.parser.xml]: Once again, what is the actual name of the constant in the extension? Does it have the typo or not?

File: reference/sybase/ini.xml [hunk 5]
```diff
@@ -53,7 +53,7 @@
      <entry></entry>
     </row>
     <row>
-     <entry>sybase.compatability_mode</entry>
+     <entry>sybase.compatibility_mode</entry>
```
Comment (nikic) [inline reference/sybase/ini.xml]: Same question here.

File: reference/yaf/yaf-route-map.xml [hunk 6]
```diff
@@ -57,7 +57,7 @@
     </fieldsynopsis>
     <fieldsynopsis>
      <modifier>protected</modifier>
-     <varname linkend="yaf-route-map.props.delimeter">_delimeter</varname>
+     <varname linkend="yaf-route-map.props.delimiter">_delimiter</varname>
```
Comment (nikic) [inline reference/yaf/yaf-route-map.xml]: Same here.

File: reference/mssql/ini.xml (hunk 2 above)
Comment (cmb69) [reply reference/mssql/ini.xml]: Both names are supported: https://github.com/php/php-src/blob/05fb10165081cb4fb55a9943b44cab62c627db7a/ext/mssql/php_mssql.c#L250-L251

File: reference/mysqlnd_qc/quickstart.xml (hunk 3 above)
Comment (cmb69) [reply reference/mysqlnd_qc/quickstart.xml]: Indeed a typo in the implementation: http://svn.php.net/viewvc/pecl/mysqlnd_qc/trunk/mysqlnd_qc_logs.c?revision=331414&view=markup#l125

File: reference/parle/parle.parser.xml (hunk 4 above)
Comment (cmb69) [reply reference/parle/parle.parser.xml]: See https://github.com/weltling/parle/issues/14

File: reference/sybase/ini.xml (hunk 5 above)
Comment (cmb69) [reply reference/sybase/ini.xml]: Implementation typo: https://github.com/php/php-src/blob/PHP-5.2/ext/sybase/php_sybase_db.c#L271

File: reference/yaf/yaf-route-map.xml (hunk 6 above)
Comment (cmb69) [reply reference/yaf/yaf-route-map.xml]: See https://github.com/laruence/yaf/commit/e366da2b99df147d66b790bdea7c982cef74875b#diff-ee82af77d5310c2a2fa9060fdcc2e16f.

File: reference/mysqlnd_qc/quickstart.xml (hunk 3 above)
Comment (carusogabriel) [reply reference/mysqlnd_qc/quickstart.xml]: Should we change for `PHP-7.4+` or is a bug fix, and should enter `PHP-7.2+`?

File: reference/sybase/ini.xml (hunk 5 above)
Comment (carusogabriel) [reply reference/sybase/ini.xml]: Should we change for `PHP-7.4+` or is a bug fix, and should enter `PHP-7.2+`?

File: reference/mssql/ini.xml (hunk 2 above)
Comment (nikic) [reply reference/mssql/ini.xml]: Okay, then I think it's fine to only mention the fixed name.

File: reference/sybase/ini.xml (hunk 5 above)
Comment (nikic) [reply reference/sybase/ini.xml]: This extension does not exist in PHP 7, so this change should just be reverted.

File: reference/mysqlnd_qc/quickstart.xml (hunk 3 above)
Comment (nikic) [reply reference/mysqlnd_qc/quickstart.xml]: The last commit to this extension is 5 years old, so I don't think it makes sense to change this.

**Comment by nikic** [comment]
@php-pulls merge

**Pull request closed without merging**